lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -171,10 +171,6 @@ def unstore(customer_vault_id, options = {})
       end
       alias_method :delete, :unstore
 
-      def supports_network_tokenization?
```

Comment:
It was always defaulted to true because there was no way to verify it before. There still isn't an "official" way, but the guidance from Braintree was to check for the specific error response coming back when attempting a transaction with a network tokenized card & cryptogram.